PRS4001-5000/PRS4285GabraKr.xml

```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
-type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" type="pers" xml:lang="en" xml:id="PRS4285GabraKr">
-   <teiHeader xml:base="https://betamasaheft.eu/">
-      <fileDesc>
-         <titleStmt>
-            <title>Gabra Krǝstos of Gunda Gunde</title>
```

Comment:
Gäbrä Kréstos
G.K. (K-:y l?FNF; d. 1594) was the seventh
leader of the Stephanite monastery of ÷Gundä
Gunde. He assumed that position in September
1572 following the death of his predecessor
÷Isayéyyas. G.K. led the monastery until he
died and was succeeded by ÷Yéshaq. At least
one manuscript (tentatively dated to the 18th
cent.) containing his gädl is known to exist, but
it has not been published.
Src.: ms. Mordini 112; Institute of Archaeology, Addis
Abäba (I.A.A.) 48.
Lit.: Antonio Mordini, “Il convento di Gunde Gundiè”,
RSE 12, 1953 [1954], 29–63, here 59; Aleksander
Ferenc, “Les Actes d’Isaïe de Gunde-Gunde”, AE 10,
1976, 243–94, here 243; KinBibl 73, no. 61.
Steven Kaplan

Reply:
So he is the one now created as PRS13661GabraKr